package.json

```diff
@@ -67,7 +67,8 @@
     "jest": "25.2.7",
     "node-pty": "0.9.0",
     "prettier": "2.0.4",
-    "ts-jest": "25.3.1"
+    "ts-jest": "25.3.1",
+    "utility-types": "^3.10.0"
```

Comment:
Can replace `type-fest`?

Reply:
Unfortunately no, type-fest has types that utility-types doesn't and vice versa. Don't think it matters on the types deps as they're devDeps anyway